lms/templates/dashboard/_dashboard_course_listing.html

```diff
@@ -454,3 +454,16 @@ <h4 class="message-title">${_('Your current verification will expire soon.')}</h
 <%static:require_module_async module_name="js/dateutil_factory" class_name="DateUtilFactory">
     DateUtilFactory.transform(iterationKey=".localized-datetime");
 </%static:require_module_async>
+
+
+<%static:require_module_async module_name="js/commerce/track_ecommerce_events" class_name="TrackECommerceEvents">
+  var link = $("#ecommerce_upgrade_link");
+  function trackEvent() {
+    TrackECommerceEvents.trackUpsellClick('course_dashboard_green', {
+      pageName: "course_dashboard",
+      linkType: "button",
+      linkCategory: "green_upgrade"
+    });
+  }
+  link.unbind('click.ecommerce_link').bind('click.ecommerce_link', trackEvent);
```

Comment:
Fixing the double js library loads on this page is out of scope for this ticket.